iml-agent/src/action_plugins/lpurge.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::agent_error::ImlAgentError;
+
+use std::fmt;
+use std::path::Path;
+
+#[derive(serde::Deserialize)]
+pub struct Config {
+    fs_name: String,
+    ost_name: String,
+    pool_name: String,
+    freelo: u8,
+    freehi: u8,
+}
+
+impl fmt::Display for Config {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "\
+             device={fs_name}-{ost_name}\n\
+             dryrun=true\n\
+             freehi={freehi}\n\
+             freelo={freelo}\n\
+             listen_socket=/run/lpurge-{fs_name}-{ost_name}-{pool_name}.sock\n\
+             max_jobs=0\n\
+             pool={fs_name}.{pool_name}\n\
+             ",
```

Comment:
Will be indents there?
Also Rust has verbatim string literals, e.g.
```
    r#"
        . . . .
        . . . .
    "#
```
In this case, however, this will contain the indentation.

Reply:
I think it will not be indented.

Reply:
AFAIK it is a recommended and documented method for unindented multiline strings. And I have seen it in output.